### PR TITLE
FAT-12628 Implement simple DI test for Gatling execution

### PR DESCRIPTION
**Pull request by JavokhirAbdullayev**
## Purpose
[FAT-12628](https://folio-org.atlassian.net/browse/FAT-12628) Implement simple DI test for Gatling execution

## Approach
add scala test class
image

### CODE REVIEW

File: common/src/main/resources/common/destroy-data.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: destroy data for tenant
     * url baseUrl
     * configure readTimeout = 90000
     * configure retry = { count: 5, interval: 5000 }
-    * callonce login admin
```
Comment (okolawole-ebsco) [inline common/src/main/resources/common/destroy-data.feature:7]: why does this need to change? This is in the common module, so we have to be careful here.

File: common/src/main/resources/common/tenant.feature [hunk 2]
```diff
@@ -4,6 +4,7 @@ Feature: Tenants
     * url baseUrl
     * configure retry = { count: 2, interval: 5000 }
     * configure readTimeout = 3000000
+    * call login admin
```
Comment (okolawole-ebsco) [inline common/src/main/resources/common/tenant.feature:7]: why this change?

File: data-import/src/test/java/org/folio/DataImportSimulation.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    constantString + randomLong
+  }
+
+  val protocol = karateProtocol(
+    "/_/proxy/tenants/{tenant}" -> Nil,
+    "/_/proxy/tenants/{tenant}/modules" -> Nil,
+    "/_/proxy/tenants/{tenant}/install" -> Nil,
+    "/data-import/uploadDefinitions" -> Nil,
+    "/data-import/uploadDefinitions/{uploadDefinitionId}" -> Nil,
+    "/finance/transactions" -> Nil,
```
Comment (okolawole-ebsco) [inline data-import/src/test/java/org/folio/DataImportSimulation.scala:24]: This endpoint called by file-upload.feature?

File: common/src/main/resources/common/destroy-data.feature (hunk 1 above)
Comment (JavokhirAbdullayev) [reply common/src/main/resources/common/destroy-data.feature:7]: because it was throwing 401 error at clean-up process

File: common/src/main/resources/common/tenant.feature (hunk 2 above)
Comment (JavokhirAbdullayev) [reply common/src/main/resources/common/tenant.feature:7]: image
 i've got these errors after running a test without these changes

Comment (JavokhirAbdullayev) [reply common/src/main/resources/common/tenant.feature:7]: I ran test against our perf rancher env

**Review by okolawole-ebsco — APPROVED** [review]

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by Maksat-Galymzhan — APPROVED** [review]
